### ISSUE TITLE
Question about no_tb item group

### ISSUE THREAD

**Issue by hoefling** [friendly-traceback/friendly-traceback#11]
It looks like the line
```py
items_groups["no_tb"].discard(items_groups["friendly_tb"])
```
doesn't affect anything. For example,
```py
import friendly_traceback as ft

def test_no_tb_does_not_contain_friendly_tb_items():
    """Verify items from ``"friendly_tb"`` are discarded in ``"no_tb"`` group."""
    no_tb_group = ft.base_formatters.items_groups["no_tb"]
    friendly_tb_group = ft.base_formatters.items_groups["friendly_tb"]
    assert not any(item in no_tb_group for item in friendly_tb_group)
```
will fail. I'm somewhat curious why since stdlib docs state
> Note, the *elem* argument to the `__contains__()`, `remove()`, and `discard()` methods may be a set. To support searching for an equivalent frozenset, a temporary one is created from *elem*.

Maybe the stdlib docs are out of date?

**Comment by aroberge** [comment]
The "no_tb" group is largely a left-over historical artifact; I have not really used it since I changed from what I previously called "verbosity levels" (labeled by integers) to this "include" method ... and, I think,  since I added the "suggest" item.

Looking at it, I think it should be replaced by:
```
for item in items_groups["friendly_tb"]:
    items_groups["no_tb"].discard(item)
```

**Comment by aroberge** [comment]
This has been fixed.

About the stdlib documentation: one cannot have a regular set inside another set as sets are not hashable. However, one can have a frozenset as an element of another set. If that is the case, then `discard` can be used with a regular set as an argument to remove a frozenset.   I had to read the explanation a few times to really understand it...

**Comment by hoefling** [comment]
> The "no_tb" group is largely a left-over historical artifact

I suspected this, stumbled over this only since the type checker spit an error. Got a bit curious whether there's a contradiction between what `mypy` reports and what the docs state. 

> About the stdlib documentation: one cannot have a regular set inside another set as sets are not hashable.

Ah! the docs talk about sets as elements of the target set! This was the key point for understanding this, thank you!

**Comment by aroberge** [comment]
Actually, after doing a quick search, I found out why I kept the no_tb
option around.

I hope that friendly_traceback will eventually be included by default in
some editor/ide.  (See for example
https://github.com/mu-editor/mu/issues/963 where this is scheduled for
milestone 1.2)

When I experimented with using ft as a syntax checker, I found that, in
some cases, the traceback should be hidden from the user.

https://github.com/friendly-traceback/friendly-traceback/blob/fc7c9a6c033f42b5c24ad9bdd37f7810e4bc3e39/friendly_traceback/editors_helpers.py#L66


This will need to be explored further if and when it is incorporated in Mu.

I should add the unit test that you had before closing this issue.
